modules/vebal/voting-list.gql

```diff
@@ -0,0 +1,11 @@
+type VotingGauge {
+    address: String!
+    chain: String!
+    isKilled: Boolean!
+    createdAt: Date!
+    relativeWeightCap: String
+}
+
+extend type Query {
```

Comment:
This is WIP, I still need to add the pool related data (I will ask some related questions soon).

Reply:
Please define this asap so we can find out what data exactly is needed.

Reply:
Also fine to just define it in json here in a comment and we can work from there. Might be easier as I think it's hard for you to determine the correct types here as you are not familiar with the types and data we have in other modules. I can then make a suggestion on the gql type returned here